Formula/pastebinit.rb

```diff
@@ -20,7 +19,7 @@ def install
     inreplace "pastebinit", "/usr/local/etc/pastebin.d", etc + "pastebin.d"
     system "docbook2man", "pastebinit.xml"
     bin.install "pastebinit"
-    etc.install "pastebin.d"
+    cp_r "pastebin.d", etc
```

Comment:
Yes, we could do a replace in a `post_install` step. But as per [pastebin's api doc](http://pastebin.com/api), unique `api_dev_key` needs be set by an user from their account settings.

My initial response to this situation was to put a caveat to inform the user to set their unique `api_dev_key` if they want use pastebin's service (As this client also provides multiple pastebin-like services). So, what do you think about inserting a caveat?